frontend/convex/utils/schemaMapper.ts

```
@@ -0,0 +1,108 @@
+      return {
+        slug: String(record.slug || ""),
+        name: String(record.name || ""),
+        region: String(record.region || ""),
+        leadingParty: String(record.leadingParty || record.leading_party || ""),
+        voteShare: Number(record.voteShare || record.vote_share || 0),
+        turnout: Number(record.turnout || 0),
+      };
+
+    case "polls":
+      // Ensure conductedAt is a valid ISO-8601 string or use current timestamp
+      const conductedAt = record.conductedAt || record.conducted_at;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)